msg.go

```diff
@@ -78,6 +78,13 @@ func NewMsgUUIDFromString(uuidString string) MsgUUID {
 	return MsgUUID{uuid}
 }
 
+type MsgFlowRef struct {
+	UUID string `json:"uuid" validate:"uuid4"`
+	Name string `json:"name"`
+}
+
+var NilMsgFlowRef = MsgFlowRef{"", ""}
```

Comment:
We don't want to serialize messages with `"flow": {"uuid": "", "name": ""}`.. should be a pointer on `Msg` which can be nil if there's no flow.

While you're changing this let's call it just `FlowReference` for simplicity and consistency with mailroom/goflow.